src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+
+const successMessage = `<div class="message">Promise was resolved!</div>`;
+const errorMessage = `<div class="message error-message">Promise was rejected!</div>`;
+
+const successHandler = (message) => {
+  body.insertAdjacentHTML('beforeend', successMessage);
+};
+const errorHandler = (message) => {
+  body.insertAdjacentHTML('beforeend', errorMessage);
+};
```

Comment:
The same issue with the `message` parameter in the `errorHandler` function. You should use this parameter instead of `errorMessage` in the `insertAdjacentHTML` method.